### PR TITLE
Details usage of `CC_SBT_TARGET_DIR`

### PR DESCRIPTION
**Pull request by gervaisb**
Add informations about `CC_SBT_TARGET_DIR` to run a specific module and
warn about conflicts with `SBT_DEPLOY_GOAL`.

### CODE REVIEW

File: runtimes/scala/scala.md [hunk 1]
```diff
@@ -77,12 +77,14 @@ application.secret=${APPLICATION_SECRET}
...
 
+Be aware that `SBT_DEPLOY_GOAL` will infer with `CC_SBT_TARGET_DIR`.
+
 ## Multi-module build
 
 If you have a single repository with multiple modules (and no top-level `stage`
 task), then you can specify which module to build with `SBT_DEPLOY_GOAL`.
 
-For instance, if you want to deploy the `service1` module, then add
+For instance, if you want to stage the `service1` module, then add
```
[the remark this answers is not in the document]
Comment (gervaisb) [reply runtimes/scala/scala.md]: OK. I will change it.

File: runtimes/scala/play-framework-2.md [hunk 2]
```diff
@@ -80,11 +80,26 @@ Redirect to HTTPS With Play
...
 ## Deploy on Clever Cloud
 
 Application deployment on Clever Cloud is via Git. Follow [these steps](/doc/clever-cloud-overview/add-application/) to deploy your application.
 
 
+## Bad root server path
+
+The error occurs when we cannot find a bin to execute your project.
+
+This may happen when you use `SBT_DEPLOY_GOAL` and `CC_SBT_TARGET_DIR`. If you do so, you should be aware that the `SBT_DEPLOY_GOAL` is relative the `/target/universal/stage` directory inside `CC_SBT_TARGET_DIR`.
```
[the remark this answers is not in the document]
Comment (gervaisb) [reply runtimes/scala/play-framework-2.md]: Yes, it is : https://github.com/CleverCloud/doc.clever-cloud.com/issues/371

**Comment by cnivolle** [comment]
@judu Should we merge this now?

**Comment by gervaisb** [comment]
@cnivolle I still have to make some changes but did not have enough time.
1. Change some words (https://github.com/CleverCloud/doc.clever-cloud.com/pull/370#discussion_r255028507) 
2. Remove reference to false bug (https://github.com/CleverCloud/doc.clever-cloud.com/pull/370#discussion_r255029001)

**Comment by cnivolle** [comment]
@gervaisb Okay then 👍

**Comment by gervaisb** [comment]
@cnivolle and @judu I finally made the changes. This can be merged from my point of view.

File: runtimes/scala/play-framework-2.md [hunk 3]
```diff
@@ -80,11 +80,24 @@ Redirect to HTTPS With Play
...
+For instance, if your Sbt project contains a `shared` and `play` module and you want to execute the `play` module, then add
+`CC_SBT_TARGET_DIR=play` in the application's environment variables.
+
+
 ## Deploy on Clever Cloud
 
 Application deployment on Clever Cloud is via Git. Follow [these steps](/doc/clever-cloud-overview/add-application/) to deploy your application.
 
 
+## Bad root server path
```
Comment (vvenance) [inline runtimes/scala/play-framework-2.md]: I would go with "Bad root server path error" as a title so it's understandable that we're talking about an error

File: runtimes/scala/scala.md [hunk 4]
```diff
@@ -77,12 +77,14 @@ application.secret=${APPLICATION_SECRET}
 By default, the deployment system runs `sbt stage`. If you want to run another
 goal, you can specify it with the `SBT_DEPLOY_GOAL` environment variable.
 
+Be aware that `SBT_DEPLOY_GOAL` will infer with `CC_SBT_TARGET_DIR`.
```
[the remark this answers is not in the document]
Comment (gervaisb) [reply runtimes/scala/scala.md]: I has some issues with my app and had the idea that it was a kind of conflict between  `SBT_DEPLOY_GOAL` and `CC_SBT_TARGET_DIR`. However it seems that I was wrong: https://github.com/CleverCloud/doc.clever-cloud.com/issues/371
I do not have time to investigate more. 
I think I removed the sentence in this PR.

File: runtimes/scala/play-framework-2.md (hunk 3 above)
Comment (vvenance) [reply runtimes/scala/play-framework-2.md]: @gervaisb can you please just change that so I can merge your PR ?

**Comment by vvenance** [comment]
thank you @gervaisb, I'm merging now 😄

**Pull request merged**